internal/store/indexes/filtering.go

```diff
@@ -5,12 +5,13 @@ import (
 	"github.com/iov-one/cosmos-sdk-crud/internal/store/iterator"
 	"github.com/iov-one/cosmos-sdk-crud/internal/store/types"
 	"github.com/iov-one/cosmos-sdk-crud/internal/util"
+	types2 "github.com/iov-one/cosmos-sdk-crud/types"
```

Comment:
Please use a more meaningful name.  The de facto standard would be to use `crudtypes`.

Reply:
Actually, `crud` would be better because then it'd be consistent with other files.

Reply:
Yeah I have to change that, it was the default refactoring name